frontend/apps/pablo/components/Organisms/pool/PoolDetails/PoolStakeForm.tsx

```diff
@@ -68,7 +76,7 @@ export const PoolStakeForm: React.FC<PoolDetailsProps> = ({
             TypographyProps: { color: "text.secondary" },
             BalanceProps: {
               title: <AccountBalanceWalletIcon color="primary" />,
-              balance: `${poolDetails.lpBalance} ${baseAsset?.symbol}/${quoteAsset?.symbol}`,
+              balance: `${poolDetails.lpBalance} ${principalAssetSymbol}`,
```

Comment:
if `principalAssetSymbol` is undefined, you will have a trailing blank space here

Reply:
@andresmechali yup, empty string is a fallback case if there is no asset, this will happen if an unsupported asset is used on the chain